docs/android/external-native-dependencies.md

```diff
@@ -1,34 +1,33 @@
 # Add a new external native dependency
 
-When we update a dependency there are no major changes as versions will be taken from [.build/package.json](../../.build/package.json) and a new [snapshot](http://trinerdis.cz:8000/repository/snapshots/com/trinerdis/skypicker/) will be deployed.
+When we update a dependency there are no major changes as versions will be taken from 
```

Comment:
I would also put somewhere, probably on this page, information about external dependencies artifacts versioning schema (the package version + RN version) with an explanation why is this schema used.